packages/pluggableWidgets/sidebar-toggle-web/src/components/Toggle.tsx

```diff
@@ -0,0 +1,74 @@
+import { createElement, CSSProperties, HTMLAttributes, ReactElement } from "react";
+import classNames from "classnames";
+import { WebIcon } from "mendix";
+import { Icon } from "@mendix/piw-utils-internal";
+
+interface ToggleProps {
+    caption?: string;
+    className: string;
+    icon?: WebIcon;
+    tabIndex?: number;
+    tooltip?: string;
+    render: "button" | "link";
+    role: string;
+    style?: CSSProperties;
+}
+
+export function Toggle(props: ToggleProps): ReactElement {
+    const commonProps = {
+        tabIndex: props.tabIndex,
+        title: props.tooltip,
+        ariaLabel: props.tooltip,
+        style: props.style
+    };
+
+    const icon = props.icon ? (
+        <Icon icon={props.icon} className="widget-sidebar-toggle-icon" fallback={<span />} />
+    ) : null;
```

Comment:
Do you not want this to be `<span />` if `props.icon` is not defined? If so, just remove the ternary and only use the `<Icon` component

Reply:
Fixed